CRM/Contribute/Form/Task/Status.php

```diff
@@ -278,6 +281,7 @@ public static function processForm($form, $params) {
         $input['trxn_id'] = $contribution->invoice_id;
       }
       $input['trxn_date'] = $params["trxn_date_{$row['contribution_id']}"] . ' ' . date('H:i:s');
+      $input['is_email_receipt'] = $is_email_receipt ? TRUE : FALSE;
```

Comment:
There's a bit of a crackdown on these types of expressions going on right now. Suggest `$input['is_email_receipt'] = !empty($params['is_email_receipt']);` and then remove line 216 above completely.

Reply:
I have fixed this